packages/components/src/components/as-histogram-widget/as-histogram-widget.tsx

```diff
@@ -365,7 +365,7 @@ export class HistogramWidget {
    * @memberof HistogramWidget
    */
   @Method()
-  public async getSelection(): Promise<number[] | string[]> {
+  public async getSelection(): Promise<Array<number | Date> | string[]> {
```

Comment:
I know typescript likes to complain about [] vs Array, but I wonder if we could unify this to Array<number | Date | string> ?

Reply:
yep!